### ISSUE TITLE
A11y_WINFORMS_DialogControls_ColorDialog_Screenreader : The NVDA is not announcing about the color in color dialog.

### ISSUE THREAD

**Issue by adbhosale** [dotnet/winforms#7352]
### .NET version

.NET Framework Version 4.8.04161
Operating System: Windows 11 Enterprise 21H2

### Did it work in .NET Framework?

No

### Issue description

GitHubTags:#A11yMAS;#A11yTCS;#.NET Core[WINFORMS]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.3.1;#DesktopApp;#Win11;#Benchmark;#Screenreader#NVDA

Actual:
 The NVDA is not announcing about the color in color dialog.

Expected:
 The NVDA should be announcing about the color in color dialog.

User Impact:
The low vision users and no vision users will get confused about the colors

### Steps to reproduce

1. Open Core testing solution in VS 2019 Int Preview
2. Right click on cs.proj file and click on Build.
3.Then run the application. 
4.Main Form screen would appear.
5.Click on DialogControls:Testing the controls under the Dialog  Tab.
6.Navigate to ColorDialog button and click on it.
7.Start NVDA and check the colors' names are announced or not.
[nvda_colors are not announced by NVDA.zip](https://github.com/dotnet/winforms/files/8989131/nvda_colors.are.not.announced.by.NVDA.zip)

**Comment by merriemcgaw** [comment]
@Olina-Zhang @vladimir-krestov what do we show for the UIA Hierarchy? What do Narrator or Jaws do? This is a Windows dialog so I doubt there's much we can do but we'll record our findings here.

**Comment by Olina-Zhang** [comment]
@merriemcgaw In Inspect: UIA tree, there isn't every color blocks, and Narrator also repro, it cannot focus on every color area.
![image](https://user-images.githubusercontent.com/26474449/176411266-455c9d44-8adc-47da-8a25-413f507eacc8.png)
![Narrator1](https://user-images.githubusercontent.com/26474449/176412027-60bd0c1b-6203-4ec7-9b47-ff1c88f7979b.gif)

**Comment by merriemcgaw** [comment]
Exception request made https://ceacc.visualstudio.com/ceaccex/_workitems/edit/130273

**Comment by merriemcgaw** [comment]
Exception approved. Closing this issue

**Comment by Tanya-Solyanik** [comment]
Closing per request from Samhitha K S (TATA CONSULTANCY SERVICES LTD)

Issue is fixed. Adding #Closed; tag.
